lib/Conversion/TorchToLinalg/Uncategorized.cpp

```diff
@@ -2876,9 +2896,15 @@ class ConvertInterpolateOp
                 [&](OpBuilder &b, Location loc, ValueRange args) {
                   Value retVal;
                   if (mode.substr(0, 7) == "nearest") {
+                    std::string coordTfMode =
+                        mode.substr(7, mode.find(",") - 7);
```

Comment:
The coordTfMode  is separated by mode via `_`, so shouldn't this be 
```suggestion
                        mode.substr(8, mode.find(",") - 8);
```

Reply:
If the coordinate tf mode isn't specified, then there won't be an underscore at the end. Starting the substring at or after npos would cause an error. We could make this a conditional or something, but I opted to just pass the underscore in with the coordTfMode (see the bilinear interpolate section above, e.g. `if (coordStr == "_asymmetric") { ...`

Reply:
@mgehre-amd Does my comment address your concern sufficiently? If so, I'd like to merge this PR.

Reply:
Ok, looks good!